lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -241,6 +241,19 @@ sub machine {
     return $self->settings_hash->{MACHINE};
 }
 
+sub obs_url() {
+    return "https://build.opensuse.org/";
+}
```

Comment:
openQA does not know, no. It gets it over the API posted. We could introduce yet another variable with a reference link though